data/ddl.sql

```diff
@@ -22,6 +22,8 @@ create table locations (
     id uuid primary key,
     country text not null,
     city text not null,
+    street_number text not null,
+    street_name text not null,
     state text,
     zip text
```

Comment:
@trojanpauro In a lot of places street numbers may not be there. Please add `address text not null`.

Reply:
@trojanpauro Remove street_number and street_name and add `address text not null`.